u1-request-response/app.js

```diff
@@ -0,0 +1,83 @@
+// Import libraries
+const express = require('express')
+
+// Setup app
+const app = express()
+
+// Middleware
+app.use(express.static('public'))
+
+// Routes
+app.get('/lab', (req, res) => {
+    res.send(`
+    <html><body>
+    <li>1. <a href="/profiles/parinya">/profiles/parinya</a></li>
+    <li>2. <a href="/api/profiles/parinya">/api/profiles/parinya</a></li>
+    <li>3. <a href="/cat-simulator">/cat-simulator</a></li>
+    <li>4. <a href="/admin/grades">/admin/grades</a> AND <a href="/api/grades">/api/grades</a> AND <a href="/boredom">/boredom</a></li>
+  </body></html>
+  
+    `)
+  })
+
+app.get('/profiles/parinya', (req, res) => {
+    res.send(`
+    <html><body>
+      <h1>Parinya Krongpha</h1>
+      <ul>
+        <li>Nickname : Chinjung</li>
+        <li>Like : Cat , Advaenture movie</li>
+        </ul>
+    </body></html>
+    `)
+})
+
+app.get('/api/profiles/parinya', (req, res) => {
+    res.json({
+      name : "Parinya Krongpha",
+      nickname : "Chinjung",
+      like : "Cat , Advaenture movie"
+      
+    })
+  })
+  
+app.get('/cat-simulator', (req, res) => {
+    res.send(`
+    <html><body>
+    <h1><li><a href="/cat-result">Pet the cat</a></li></h1>
+    <head><link rel="stylesheet" href="/styles.css"></head>
+<body class="cats">    
```

Comment:
คุณไม่สามารถมีแท็ก HTML `<body>` 2 แท็กได้ นี่ควรเป็น:
You can't have 2 HTML `<body>` tags. This should be:
```js
res.send(`
  <html>
    <head><link ...></head>
    <body>
      <h1>...</h1>
      <img ... />
    </body>
  </html>
`)
```